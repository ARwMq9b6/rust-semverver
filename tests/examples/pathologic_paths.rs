#![recursion_limit="128"]

pub mod old {

}

pub mod new {
    macro_rules! blow {
        () => {};
        (_ $($rest:tt)*) => {
            pub mod a { blow!($($rest)*); }
            pub mod b { pub use super::a::*; }
        }
    }

    blow!(_ _ _ _ _ _ _ _
          _ _ _ _ _ _ _ _
          _ _ _ _ _ _ _ _
          _ _ _ _ _ _ _ _
          _ _ _ _ _ _ _ _
          _ _ _ _ _ _ _ _
          _ _ _ _ _ _ _ _
          _ _ _ _ _ _ _ _);
}
