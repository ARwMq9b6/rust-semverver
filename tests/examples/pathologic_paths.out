version bump: 1.0.0 -> (TechnicallyBreaking) -> 1.1.0
warning: Addition of at least one path to `a` (TechnicallyBreaking)
  --> $REPO_PATH/tests/examples/pathologic_paths.rs:11:25
   |
11 |             pub mod a { blow!($($rest)*); }
   |                         ^^^^^^^^^^^^^^^^^^^

warning: Addition of at least one path to `b` (TechnicallyBreaking)
  --> $REPO_PATH/tests/examples/pathologic_paths.rs:12:25
   |
12 |             pub mod b { pub use super::a::*; }
   |                         ^^^^^^^^^^^^^^^^^^^^^^

